.github/workflows/docs.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - uses: actions-rs/cargo@v1
```

Comment:
@GuillaumeGomez You can substitute the invocations of `gir` and `rustdoc-stripper` above with `./generator.py --embed-docs` now :)

Reply:
Not working.

Reply:
It does, you need a bit more than calling the generator here; see what I did at https://github.com/gtk-rs/gtk4-rs/pull/313

Reply:
@GuillaumeGomez Have you deleted `generator.py` and symlinked it to `./gir/generator.py`?

Reply:
(If the error message were wrong, blame `argparse` :smirk:)

Reply:
Ah, I didn't!